ntlm.go

```diff
@@ -86,6 +89,20 @@ func utf16le(val string) []byte {
 	return v
 }
 
+func decodeUTF16le(val []byte) (string, error) {
```

Comment:
The function you want is ucs22str I think. If this is different, then add a comment explaining the differences.